.github/CODEOWNERS

```
@@ -0,0 +1,10 @@
+# This CODEOWNERS file sets the individuals responsible for code in the era-test-node repository.
+
+# These users are the default owners for everything in the repo.
+# They will be requested for review when someone opens a pull request.
+* @JackHamer09
```

Please replace specific names with the GitHub team
 ```suggestion
* @matter-labs/devxp
```